test/check-machines-create

```diff
@@ -1482,6 +1485,18 @@ vnc_password= "{vnc_passwd}"
                 self._deleteVm(dialog) \
                     .checkEnvIsEmpty(dialog)
 
+        def createAndImportTooltipsTest(self):
+            b = self.browser
+
+            b.wait_visible("#import-existing-vm:not(:disabled)")
+            b.mouse("#import-existing-vm", "mouseenter")
+            b.wait_in_text("#import-button-tooltip", "Create virtual machine by importing a disk image of an existing VM installation")
```

Comment:
Feels a bit excessive to repeat the whole string here -- checking for "existing" seems enough, and likewise for perhaps "medium" for the create-new one.

Reply:
We need to decide whether we want to say "VM" or "virtual machine" in the create and import buttons and be consistent.

```suggestion
            b.wait_in_text("#import-button-tooltip", "Import disk image of an existing installation to create a new virtual machine")
```

Reply:
FTR, the string change needs to go into the test, not the code. That's why I said the test shouldn't repeat the long string, just check for a significant word/short phrase.